packages/integrations/src/interfaces/firewall-summary/firewall-summary-types.ts

```diff
@@ -0,0 +1,31 @@
+export interface FirewallInterfacesSummary {
+  data: FirewallInterface[];
+  timestamp: Date;
+}
+
+export interface FirewallInterface {
+  name: string;
+  recv: number;
+  trans: number;
+}
+
+export interface FirewallVersionSummary {
+  version: string;
+}
+
+export interface FirewallCpuSummary {
+  total: number;
+  idle: number;
+  system: number;
+  user: number;
+}
+
+export interface FirewallMemorySummary {
+  total: number;
+  user: number;
+  nice: number;
+  system: number;
+  intr: number;
+  idle: number;
+  percent: number;
+}
```

Comment:
What are `intr` and `nice`? Please rename if possible to something more understandable